tidb-cloud/use-chat2query-knowledge.md

```diff
@@ -0,0 +1,218 @@
+
+## Before you begin
+
+Before creating a knowledge base for your database, make sure that you have the following:
+
+- A [Chat2Query Data App](/tidb-cloud/use-chat2query-api.md#1-create-a-chat2query-data-app)
+- An [API key for the Chat2Query Data App](/tidb-cloud/use-chat2query-api.md#create-an-api-key)
+
+## Step 1. Create a knowledge base for the linked database
+
+> **Note:**
+>
+> The knowledge used by Chat2Query is **structured according to the database dimension**. You can connect multiple Chat2Query Data Apps to the same database, but each Chat2Query Data App can only use knowledge from a specific database it is linked to.
+
+In your Chat2Query Data App, you can create a knowledge base for a specific database by calling the `/v3/knowledgeBases` endpoint. After creation, you will get a `knowledge_base_id` for future knowledge management.
+
+The following is a general code example for calling this endpoint.
+
+> **Tip**
+>
+> To get a specific code example for your endpoint, click the endpoint name in the left pane of your Data App, and then click **Show Code Example**. For more information, see [Get the example code of an endpoint](/tidb-cloud/use-chat2query-api.md#get-the-example-code-of-an-endpoint).
+
+```bash
+curl --digest --user ${PUBLIC_KEY}:${PRIVATE_KEY} --request POST 'https://<region>.data.tidbcloud.com/api/v1beta/app/chat2query-<ID>/endpoint/v3/knowledgeBases'\
+ --header 'content-type: application/json'\
+ --data-raw '{
+    "cluster_id": "<The ID of the cluster to which the database belongs>",
+    "database": "<The name of the target database>",
+    "description": "<Your knowledge base description>"
+}'
+```
+
+An example response is as follows:
+
+```json
+{
+    "code":200,
+    "msg":"",
+    "result":
+        {
+            "default":true,
+            "description":"",
+            "knowledge_base_id":2
+        }
+}
+```
+
+After getting the response, record the `knowledge_base_id` value in your response for later use.
+
+## Step 2. Choose a knowledge type
+
+The knowledge base of each database can contain multiple types of knowledge. Before adding knowledge to your knowledge base, you need to choose a knowledge type that best suits your use case.
+
+Currently, Chat2Query knowledge bases support the following knowledge types. Each type is specifically designed for different scenarios and has a unique knowledge structure.
+
+- Few-shot example
+- Term-sheet explanation
+- Instruction
+
+### Few-shot example
+
+Few-shot example refers to the Q&A learning samples provided to Chat2Query, which include sample questions and their corresponding answers. These examples help Chat2Query handle new tasks more effectively.
+
+> **Note:**
+>
+> Make sure the accuracy of newly added examples, because the quality of examples affects how well Chat2Query learns. Poor examples, such as mismatched questions and answers, can degrade Chat2Query performance on new tasks.
```

Comment:
```suggestion
> Make sure the accuracy of newly added examples, because the quality of examples affects how well Chat2Query learns. Poor examples, such as mismatched questions and answers, can degrade the performance of Chat2Query on new tasks.
```